app/experimenter/experiments/tests/api/v1/test_serializers.py

```diff
@@ -150,11 +150,8 @@ def test_serializer_outputs_expected_schema(self):
                 "results_confidence": None,
                 "results_measure_impact": None,
                 "results_impact_notes": None,
+                "results_impact_notes": None,
```

Comment:
Oops duplicate line!

Reply:
Oh okay sorry didn't see that, will fix it